data/data/config.tf

```diff
@@ -37,6 +37,30 @@ EOF
 
 }
 
+variable "bootstrap_enabled" {
```

Comment:
these are best kept in the `variables-gcp.tf` as they don't make whole lot of sense globally yet.

Reply:
I was thinking libvirt already can use these. I have no problem moving them to variables-gcp, does that mean I should rename them to gcp_*?

Reply:
> I was thinking libvirt already can use these. 

yeah, but i think let's keep it gcp only and then standardize on the boostrap destroy moving ahead when more platforms need things like these.

> I have no problem moving them to variables-gcp, does that mean I should rename them to gcp_*?

i'm fine with either.